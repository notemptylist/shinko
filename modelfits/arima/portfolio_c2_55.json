{
    "numlags": 400,
    "results": [
        {
            "mean": 0.0026768311059989563,
            "order": [
                0,
                0,
                0
            ],
            "rmse": 3.8029770862402894,
            "tstamp": 1639938870.0577528
        },
        {
            "mean": 0.0026768311059989563,
            "order": [
                3,
                0,
                0
            ],
            "rmse": 3.803167296575381,
            "tstamp": 1639938870.0577528
        },
        {
            "mean": 0.0026768311059989563,
            "order": [
                2,
                2,
                0
            ],
            "rmse": 6.1913386783172095,
            "tstamp": 1639938870.0577528
        }
    ],
    "stream": "portfolio_c2_55.json",
    "todo": [
        [
            0,
            0,
            1
        ],
        [
            0,
            0,
            2
        ],
        [
            0,
            1,
            0
        ],
        [
            0,
            1,
            1
        ],
        [
            0,
            1,
            2
        ],
        [
            0,
            2,
            0
        ],
        [
            0,
            2,
            1
        ],
        [
            0,
            2,
            2
        ],
        [
            1,
            0,
            0
        ],
        [
            1,
            0,
            1
        ],
        [
            1,
            0,
            2
        ],
        [
            1,
            1,
            0
        ],
        [
            1,
            1,
            1
        ],
        [
            1,
            1,
            2
        ],
        [
            1,
            2,
            0
        ],
        [
            1,
            2,
            1
        ],
        [
            1,
            2,
            2
        ],
        [
            2,
            0,
            0
        ],
        [
            2,
            0,
            1
        ],
        [
            2,
            0,
            2
        ],
        [
            2,
            1,
            0
        ],
        [
            2,
            1,
            1
        ],
        [
            2,
            1,
            2
        ],
        [
            2,
            2,
            1
        ],
        [
            2,
            2,
            2
        ],
        [
            3,
            0,
            1
        ],
        [
            3,
            0,
            2
        ],
        [
            3,
            1,
            0
        ],
        [
            3,
            1,
            1
        ],
        [
            3,
            1,
            2
        ],
        [
            3,
            2,
            0
        ],
        [
            3,
            2,
            1
        ],
        [
            3,
            2,
            2
        ],
        [
            6,
            0,
            0
        ],
        [
            6,
            0,
            1
        ],
        [
            6,
            0,
            2
        ],
        [
            6,
            1,
            0
        ],
        [
            6,
            1,
            1
        ],
        [
            6,
            1,
            2
        ],
        [
            6,
            2,
            0
        ],
        [
            6,
            2,
            1
        ],
        [
            6,
            2,
            2
        ]
    ],
    "version": "0.1"
}