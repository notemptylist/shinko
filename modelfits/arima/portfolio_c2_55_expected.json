{
    "numlags": 400,
    "results": [
        {
            "mean": 0.1609402976331557,
            "order": [
                1,
                1,
                1
            ],
            "rmse": 0.3684769195634832,
            "tstamp": 1640126129.8519688
        },
        {
            "mean": 0.1609402976331557,
            "order": [
                6,
                0,
                0
            ],
            "rmse": 0.37616286630017254,
            "tstamp": 1640126129.8519688
        },
        {
            "mean": 0.1609402976331557,
            "order": [
                2,
                1,
                0
            ],
            "rmse": 0.4220182252963796,
            "tstamp": 1640126129.8519688
        },
        {
            "mean": 0.028685712029200113,
            "order": [
                6,
                1,
                1
            ],
            "rmse": 0.3450740980759673,
            "tstamp": 1640310610.2789512
        },
        {
            "mean": 0.028685712029200113,
            "order": [
                5,
                0,
                3
            ],
            "rmse": 0.347208652475499,
            "tstamp": 1640310610.2789512
        },
        {
            "mean": 0.028685712029200113,
            "order": [
                5,
                1,
                0
            ],
            "rmse": 0.3793431265120785,
            "tstamp": 1640310610.2789512
        }
    ],
    "stream": "portfolio_c2_55_expected.json",
    "todo": [
        [
            0,
            0,
            0
        ],
        [
            0,
            0,
            1
        ],
        [
            0,
            0,
            3
        ],
        [
            0,
            1,
            0
        ],
        [
            0,
            1,
            1
        ],
        [
            0,
            1,
            3
        ],
        [
            0,
            3,
            0
        ],
        [
            0,
            3,
            1
        ],
        [
            0,
            3,
            3
        ],
        [
            1,
            0,
            0
        ],
        [
            1,
            0,
            1
        ],
        [
            1,
            0,
            3
        ],
        [
            1,
            1,
            0
        ],
        [
            1,
            1,
            3
        ],
        [
            1,
            3,
            0
        ],
        [
            1,
            3,
            1
        ],
        [
            1,
            3,
            3
        ],
        [
            2,
            0,
            0
        ],
        [
            2,
            0,
            1
        ],
        [
            2,
            0,
            3
        ],
        [
            2,
            1,
            1
        ],
        [
            2,
            1,
            3
        ],
        [
            2,
            3,
            0
        ],
        [
            2,
            3,
            1
        ],
        [
            2,
            3,
            3
        ],
        [
            3,
            0,
            0
        ],
        [
            3,
            0,
            1
        ],
        [
            3,
            0,
            3
        ],
        [
            3,
            1,
            0
        ],
        [
            3,
            1,
            1
        ],
        [
            3,
            1,
            3
        ],
        [
            3,
            3,
            0
        ],
        [
            3,
            3,
            1
        ],
        [
            3,
            3,
            3
        ],
        [
            4,
            0,
            0
        ],
        [
            4,
            0,
            1
        ],
        [
            4,
            0,
            3
        ],
        [
            4,
            1,
            0
        ],
        [
            4,
            1,
            1
        ],
        [
            4,
            1,
            3
        ],
        [
            4,
            3,
            0
        ],
        [
            4,
            3,
            1
        ],
        [
            4,
            3,
            3
        ],
        [
            5,
            0,
            0
        ],
        [
            5,
            0,
            1
        ],
        [
            5,
            1,
            1
        ],
        [
            5,
            1,
            3
        ],
        [
            5,
            3,
            0
        ],
        [
            5,
            3,
            1
        ],
        [
            5,
            3,
            3
        ],
        [
            6,
            0,
            1
        ],
        [
            6,
            0,
            3
        ],
        [
            6,
            1,
            0
        ],
        [
            6,
            1,
            3
        ],
        [
            6,
            3,
            0
        ],
        [
            6,
            3,
            1
        ],
        [
            6,
            3,
            3
        ]
    ],
    "version": "0.1"
}